drivers/media/i2c/ov7251.c

```diff
@@ -309,9 +312,8 @@ static const struct reg_value ov7251_setting_vga[] = {
 	{ 0x3c00, 0x89 },
 	{ 0x3c01, 0x63 },
 	{ 0x3c02, 0x01 },
-	{ 0x3c03, 0x00 },
 	{ 0x3c04, 0x00 },
-	{ 0x3c05, 0x03 },
+	{ 0x3c05, 0x01 },
```

Comment:
What does this change do?

Reply:
{0x3C04, 0x3C05} number of active frames

0x3C03 when 0x17: External trigger snapshot mode A rising edge on FSIN pin wakes the sensor up and streams out {0x3404, 0x3405} frames

So previously each rising edge triggered 3 frames (the default for some reason), which sort of defeats the point of it being solely externally triggered.
https://datasheetspdf.com/pdf-file/1252891/OmniVision/OV7750/1

Reply:
Understood.